packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -488,6 +488,23 @@ export const defaultErrorMessages = {
     });
   },
 
+  'consistent-callback-args': (testInfo: IsConformantOptions, invalidProps: string[]) => {
+    const { displayName } = testInfo;
+    const { testErrorInfo, resolveInfo } = errorMessageColors;
+
+    return getErrorMessage({
+      displayName,
+      overview: 'uses non-standard callback arguments.',
+      details: ['These callback(s) need have two params:', testErrorInfo(formatArray(invalidProps))],
+      suggestions: [
+        `Ensure that callbacks ${resolveInfo(displayName + `'s`)} have two params (an event and data object)`,
+        `If the name is correct, add the prop to isConformant ${resolveInfo(
+          "testOptions['consistent-callback-args'].ignoreProps",
```

Comment:
This does not make sense to me, what is `name`

Reply:
@ecraig12345 proposed nice change, now it's clearer